recipes/stages/_base_/data/coco_ote_yolox.py

```diff
@@ -0,0 +1,45 @@
+_base_ = [
+    './data.py',
+    './pipelines/coco_ote_yolox_pipeline.py'
+]
+
+__dataset_type = 'CocoDataset'
+__data_root = 'data/coco/'
+
+__train_pipeline = {{_base_.train_pipeline}}
+__test_pipeline = {{_base_.test_pipeline}}
+__img_scale = {{_base_.img_scale}}
+
+__samples_per_gpu = 2
+
+data = dict(
+    samples_per_gpu=__samples_per_gpu,
+    workers_per_gpu=4,
+    num_classes=2,
```

Comment:
Are other templates also using 2 as the default value of num_classes?

Reply:
By coco_ote.py, yes. But as I mentioned above we should think about definition of __samples_per_gpu